web/modules/webspark/webspark_blocks/js/anchor-menu.js

```diff
@@ -10,6 +10,13 @@
 
         let section = $('.uds-anchor-menu-wrapper h4').text().toLowerCase().trim();
 
+        // If the user is an admin, we clear the anchor menu items to not duplicate links
+        if(drupalSettings.is_admin) {
```

Comment:
1. Formatting: can we get a space between the `if` and the `(` so we match coding standards.
2. I didn't test for this, but does this issue affect any user who can access this page/ui or only Admins? I'm trying to understand if this issue only affects admins or anyone who can access this page. (I tested other things, but not that).

Reply:
The problem only occurs in Layout Builder, so non-admin users would not be affected.